linux/files/modprobe.conf.jinja

```diff
@@ -0,0 +1,12 @@
+{% if module_content.disabled is defined and module_content.disabled -%}
+blacklist {{ module_name }}
+{%- else -%}
+
+{%- if module_content.option is defined -%}
```

Comment:
is defined conditions are not needed if you'll use eg. `module_content.get('option', {})`